app/subscriber/src/features/my-reports/utils/isAutoSend.ts

```diff
@@ -1,6 +1,7 @@
 import { IReportModel } from 'tno-core';
 
 export const isAutoSend = (report: IReportModel) => {
+  if (report.events === undefined) return;
```

Comment:
Why return `undefined`?

Reply:
Not sure if this was a weird one off in my local, but I was getting an error because report.events was undefined when the below was trying to execute

Reply:
Then return false so that the function `isAutoSend` is a good function.